src/core/i18n/utils.js

```diff
@@ -181,6 +181,16 @@ export function makeMomentLocale(locale) {
   return locale.replace('_', '-').toLowerCase();
 }
 
+// Functionality based on oneLine form declandewet/common-tags https://goo.gl/4PzaJI
+// If this function is altered muffinresearch/babel-gettext-extractor also needs
+// to be updated.
+function oneLineTranslationString(str) {
```

Comment:
I'm guessing this was copy/pasted and we can't use `oneLine` directly.

Can we change the variable name to `string` or `i18nString`? I'm just wary of no vowel variables.